release_notes/ocp-4-7-release-notes.adoc

```diff
@@ -706,6 +706,15 @@ The machine health resource example described in xref:../machine_management/depl
 
 Starting with {product-title} 4.7, during power-based remediation if the power operations do not complete successfully, the bare-metal machine controller triggers the reprovisioning of the unhealthy node. The exception to this is if the node is a master node or a node that was provisioned externally.
 
+[id="cp-4-7-ipi-install-diagnosing-duplicate-mac-address"]
+==== Diagnosing a duplicate MAC address when provisioning a new host in the cluster
+
+If the MAC address of an existing bare-metal node in the cluster matches the MAC address of a bare-metal host you are attempting to add to the cluster, a registration error is displayed for the failed bare-metal host.
```

Comment:
@aireilly This might be my ignorance. Do you need to mention what is failing and/or when this situation might occur? 

Something like:
_When provisioning a new node into an existing bare metal cluster, if the MAC address of an existing bare-metal node in the cluster matches the MAC address of a bare-metal host you are attempting to add to the cluster, the installation fails and a registration error is displayed for the failed bare-metal host._

Reply:
Makes sense. I updated with your suggestion :)